feat(wld): update daily volumn holdstation agg

dev-holdstation opened the pull request.

noateden commented on aggregators/holdstation-agg/index.ts:
```diff
@@ -41,6 +45,37 @@ const fetch = async (options: FetchOptions) => {
     dailyFees.addToken(feeToken, log.feeAmount)
   })
 
+  const provider = getProvider(CHAIN.WC);
```
this is too heavy to do, is there other way to do?

dev-holdstation replied:
i'm sorry that our ABI does not emit necessary events for getting logs. I'm looking for alternative ways to approach. Could you kindly suggest some solutions for us?

dev-holdstation commented:
thanks, @noateden. We added another router to our agg, so this change is for that (sum the vol of 3 event + vol of added router). The original method for added router filtered all WLD txs by function, — so now we’ve switched based on your suggestion. This is still agg, not dex

llamabutler commented:
The holdstation-agg adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators holdstation-agg
    
    🦙 Running HOLDSTATION-AGG adapter 🦙
    ---------------------------------------------------
    Start Date:	Thu, 19 Jun 2025 00:43:19 GMT
    End Date:	Fri, 20 Jun 2025 00:43:19 GMT
    ---------------------------------------------------
    
    WC 👇
    Backfill start time: 16/4/2025
    Daily volume: 29.91 k
    Daily fees: 5.46 k
    End timestamp: 1750380198 (2025-06-20T00:43:18.000Z)

dev-holdstation commented:
hi @noateden, we've added logs from swapped event. Could you pls review and let me know if any issues. Thank you 😊
